use-timescale/fork-services.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Fork services
+excerpt: With developer forks, spin up a branch, run your test, throw it away, or merge it back.
+products: [cloud]
+price_plans: [performance, scale, enterprise]
+keywords: [backups, restore]
+tags: [recovery, failures]
+---
+
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+import CLIFORKS from "versionContent/_partials/_devops-cli-service-forks.mdx";
+
+# Fork $SERVICE_SHORTs
+
+Modern development is highly iterative. Developers and AI agents need safe spaces to test changes before deploying them
+to production. Forkable $SERVICE_SHORTs make this natural and easy. Spin up a branch, run your test, throw it away, or
+merge it back.
+
+A fork is an exact copy of a $SERVICE_SHORT at a specific point in time, with its own independent data and configuration,
+including:
+- The database data and schema
+- Configuration
+- An admin `tsdbadmin` user with a new password
+
+Forks are fully independent. Changes to the fork don't affect the parent $SERVICE_SHORT. You can query
+them, run migrations, add indexes, or test new features against the fork without affecting the original service.
+
+Forks are a powerful way to share production-scale data safely. BI and data science teams often need access to real
+datasets to build models or generate insights. With forkable $SERVICE_SHORTs, you easily create fast, zero-copy
+branches of a production $SERVICE_SHORT that are isolated from production, but contain all the data needed for
+analysis. You can share this fork with your analytics teams in seconds. This dramatically reduces friction getting insights
+from live data. 
```

Comment:
"seconds" is a tad optimistic imo. yes on fluid storage you might get it in 30-40 seconds right now but it's a docs page not an ad so I think we can be a bit more informational in tone.

I also wouldn't focus so much on analytics teams forks are especially useful for testing operations that might require writes (feature development that requires new columns, migrations and new indices) a BI team is often better served by a simple read replica I think.